### ISSUE TITLE
Option suggestion

### ISSUE THREAD

**Issue by cyahega** [KIRAKIRA-DOUGA/KIRAKIRA-Cerasus#126]
I have an idea that we can add an option while writing in the title, description, danmaku comment and comment section 
The option is that users can write some special character like below (an example in Japanese keyboard)
![Screenshot_20240111-160843_Chrome](https://github.com/KIRAKIRA-DOUGA/KIRAKIRA-Cerasus/assets/114506758/931f844a-9b67-4157-a3b7-fed943e6386f)
I find this is commonly used in many Chinese/Japanese video
So imagine that users can use them to make the title, description, danmaku comment and comment section look even more interesting

**Comment by cfdxkk** [comment]
Now we hava a kaomoji selecter box, perhaps we can allow users to add some custom kaomoji or symbols?
but we also need to think about the user abuse problem


也许我们可以让用户在颜文字选择框里上传自定义字符？
（但是也要考虑用户可以把不文明内容作为模板，然后到处刷屏）
@Aira-Sakuranomiya @otomad 


edit: kamoji → kaomoji

**Comment by otomad** [comment]
我觉得颜文字、绘文字、特殊符号就应该是三个不同的东西，应该分门别类地存在。

I think kaomoji, emoji, symbols are different, should exist in different categories.

**Comment by cyahega** [comment]
True, and i think it will be more convenient for users to use them
